test/system/helpers.bash

```diff
@@ -1139,5 +1139,30 @@ function sleep_to_next_second() {
     sleep 0.$(printf '%04d' $((10000 - 10#$(date +%4N))))
 }
 
+function wait_for_command_output() {
+    local cmd="$1"
+    local want="$2"
+    local tries=20
+    local sleep_delay=0.5
+
+    case "${#*}" in
+        2) ;;
+        4) tries="$3"
+           sleep_delay="$4"
+           ;;
+        *) die "Internal error: 'wait_for_command_output' requires two or four arguments" ;;
+    esac
+
+    while :; do
+        run $cmd
```

Comment:
Please don't forget `echo "$_LOG_PROMPT $cmd"` followed by `echo "$output"`. Your future self will be forever grateful.

Reply:
The prints looked a bit excessive, but I understand your point. They are now added, thanks